### ISSUE TITLE
refactor telnet option negotiation constants

### ISSUE THREAD

**Issue by jquast** [ianepperson/telnetsrvlib#4]
Nearly all of the custom-coded telnet options you program in telnetsrvlib.py can be imported from the standard python module, telnetlib, and make the beginning of your program much briefer and less daunting to a first-opener.

See for example https://github.com/jquast/x84/blob/master/x84/telnet.py#L47

**Comment by ianepperson** [comment]
That module was removed in Python 3.13. Good idea, but obsolete.

**Comment by jquast** [comment]
Yes, I maintain it in in my asyncio telnet client, server, and protocol library. https://pypi.org/project/telnetlib3/
